tox.ini

```diff
@@ -1,10 +1,13 @@
 [tox]
-envlist = py33,py34,py35,py36,pyflakes
+envlist = py27,py33,py34,py35,py36,pyflakes
 skip_missing_interpreters = true
 
 [testenv]
 commands =
     {envpython} -bb setup.py test {posargs}
+deps =
```

Comment:
I had trouble running it locally without the deps here, but not sure if I was doing something wrong.